primeqa/tableqa/trainer/tableqa_trainer.py

```diff
@@ -0,0 +1,10 @@
+from transformers import Trainer
+class TableQATrainer(Trainer):
+    def __init__(self, *args, eval_examples=None, post_process_function=None, **kwargs):
+        """Basic tableQA trainer which extends huggingface's transformers Trainer class
+
+        Args:
+            eval_examples (_type_, optional): _description_. Defaults to None.
+            post_process_function (_type_, optional): _description_. Defaults to None.
+        """
+        super().__init__(*args, **kwargs)
```

Comment:
What is the reason for creating the TableQATrainer class.? Why not use the HF trainer directly ?

Reply:
Going further when we add support for numerical questions, we will need to overwrite some of the default HF Trainer methods.